factory/mock/components/configs.go

```diff
@@ -0,0 +1,229 @@
+package components
```

Comment:
also this, can be moved in the testscommon/components

Reply:
:+1:

Reply:
moved